### DISCUSSION TITLE
No search managers registered

### DISCUSSION THREAD

**Discussion by God0nlyKnows** [topi314/LavaSearch#5] [Q&A] [answered]
Environment:
Lavalink Version: 4.0.3
Plugin: lavasearch-plugin
Deployment: docker

Upon sending a request to /v4/loadsearch, the server responds with a 500 error, and the logs indicate "No search managers registered".

but i don't know it's a problem with plugin itself or lavalink can't provide search manager config 

[trace.txt](https://github.com/topi314/LavaSearch/files/14471649/trace.txt)

**Comment by topi314** [comment, marked as answer]
LavaSearch does not come with any search managers itself
LavaSrc provides some as an example

**Reply by God0nlyKnows** [reply]
ah my bad, I had a typo in application.yml and config for lavasrc wasn't working 😅
